rclcpp_action/include/rclcpp_action/create_server.hpp

```diff
@@ -31,13 +31,19 @@
 
 namespace rclcpp_action
 {
+
```

Comment:
nitpick: you have an empty line after this `{` but not before the closing `}`, maybe just leave this newline out?